python/artm/batches_utils.py

```diff
@@ -70,8 +73,14 @@ def __init__(self, batches=None, collection_name=None, data_path='', data_format
                                        and if data_weight is list - automatically set to False
         :param class_ids: list of class_ids or single class_id to parse and include in batches
         :type class_ids: list of str or str
+        :param bool process_in_memory: process batches from disk or from RAM \
+                                       (only if data_format == Batches), model parameter is required
```

Comment:
Well, yes maybe. I will also create a page into documentation about this case, I think it'd be enough